.github/workflows/daily.yml

```diff
@@ -189,6 +192,9 @@ jobs:
       (github.event_name == 'workflow_dispatch' ||
       (github.event_name == 'schedule' && github.repository == 'redislabs/redisraft')) && !contains(github.event.inputs.skipjobs, 'tls')
     timeout-minutes: 14400
+    strategy:
+      matrix:
+        tls_mode: [ "None", "File", "Dir", "Both" ]
```

Comment:
I think running whole suite with different TLS configurations doesn't add much value because it is still just TLS :) After loading the certificates/keys and connecting to each other, rest will be same anyway.

 I think we just want to test if we can configure TLS with a file, a directory or both. So, I suggest adding "dir" and "both" cases as a single test case. Maybe, we can pass a parameter to `cluster.create(tls_cacert_location)` and depending on this parameter, we can use `dir` or `file` or `both`.

So, something like:

```python
# Skip if not TLS
test_tls_cacert_dir(cluster):
   cluster.create(3, tls_cacert_location=dir)
   cluster.execute("set x 1")
   assert cluster.execute("get x") == 1

# Skip if not TLS
test_tls_cacert_both(cluster):
   cluster.create(3, tls_cacert_location=both)
   cluster.execute("set x 1")
   assert cluster.execute("get x") == 1
```

Reply:
If `dir` will be used more, maybe we can make `dir` as the default option here and add single test cases for `file` and `both` cases.

Reply:
I like as much as possible tests, but sure, can have a small dedicated test only to verify connection creation.